src/main/java/uk/gov/companieshouse/account/validator/service/maintenance/AccountMaintenanceService.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.account.validator.service.maintenance;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.account.validator.model.validation.RequestStatus;
+import uk.gov.companieshouse.account.validator.repository.RequestStatusRepository;
+import uk.gov.companieshouse.account.validator.service.file.transfer.FileTransferStrategy;
+import uk.gov.companieshouse.logging.Logger;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * An implementation of the AccountMaintenanceService to maintain company's account files
+ */
+@Service
+public class AccountMaintenanceService implements AccountMaintenanceStrategy {
+
+    private final FileTransferStrategy fileTransferStrategy;
+    private final Logger logger;
+    private final RequestStatusRepository statusRepository;
+
+
+    @Autowired
+    public AccountMaintenanceService(Logger logger, FileTransferStrategy fileTransferStrategy, RequestStatusRepository statusRepository){
+        this.logger = logger;
+        this.fileTransferStrategy = fileTransferStrategy;
+        this.statusRepository = statusRepository;
+    }
+
+    @Override
+    public void deleteFiles() {
```

Comment:
I think a name like `deleteCompleteSubmissions` would better describe the action of the method as it not only delete's files but also the submission fromt the database. It also says which ones are deleted.

Reply:
Yes, I agree with the proposed method name.

Reply:
method name is update as per suggestion - deleteCompleteSubmissions